flow/util/autotuner/flowautotuner_hyperopt.py

```diff
@@ -257,11 +257,10 @@ def read_config():
         metric="minimum",
         mode="min",
         search_alg=algo,
-        name="ibex-test-hyperopt",
+        name="gcd-test-hyperopt",
```

Comment:
can this be synthesized from the design name, eg

name='{}-test-hyperopt'.format(design),